### ISSUE TITLE
Display name vs givenName and familyName

### ISSUE THREAD

**Issue by csarven** [ceurws/ceur-make#38]
Currently we use display name for the author. Should we move or also incorporate givenName and familyName specifically? It would mean that the toc needs to have a field for it. Another thing to investigate: is easychair metadata making that distinction or only giving display name?

**Comment by clange** [comment]
Good point. EasyChair does distinguish, well, they call it "first name" and "last name", but we should take the liberty to interpret it as "given name" and "family name". (However, according to my experience, a few authors do mix up the order, which requires manual fixing.) Implementing support for this requires

1. supporting the respective syntax of `README_EASYCHAIR` in easychair2xml.pl
1. inventing some new toc.xml markup for given name and family name
1. translating them into the right FOAF annotations in toc2ceurindex.xsl
